src/masonite/foundation/HttpKernel.py

```diff
@@ -10,13 +10,24 @@
     EncryptCookies,
 )
 from ..routes import RouteCapsule, Route
-import pydoc
+from ..middleware import APIAuthenticationMiddleware, ResponseMiddleware
+from ..utils.structures import load
 
 
 class HttpKernel:
 
-    http_middleware = []
-    route_middleware = {"web": [EncryptCookies, SessionMiddleware, VerifyCsrfToken]}
+    http_middleware = [ResponseMiddleware]
+
+    route_middleware = {
```

Comment:
What about using the API session based ? I am a bit lost with the guards versus middleware roles in here 🤔 
But as I was saying in the #45 issue, I guess it would be cool to be able to use the API and authenticate requests with:
- session (stored in cookie)
- JWT tokens

Maybe we are not naming things the same way but here I have the feeling that it's not currently possible with this SessionMiddleware only in "web" route stack